ext/kv/README.md

```diff
@@ -71,6 +73,10 @@ An HTTP `Authorization` header in the format `Bearer <ephemeral-token>` must be
 included in all requests to the data plane. The value of `<ephemeral-token>` is
 the `token` field from the metadata exchange response.
 
+An HTTP header `x-deno-database-id` (or `x-transaction-domain-id` in v1) must be
```

Comment:
Should be `x-denokv-database-id`, based on https://github.com/denoland/denokv/blob/denokv/proto/kv-connect.md

Reply:
Good catch - fixed